feat: precondition guides (#91)
sapphiredev/website: docs/Guide/preconditions/nsfw-filter.mdx

---
sidebar_position: 5
title: Locking commands to NSFW channels
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

Sometimes it can be necessary to lock certain commands to NSFW (not safe for work) channels. This can be simply achieved

> by adding the `nsfw` option to the command.

Reviewer: ```suggestion
by setting the `nsfw` option in the command!
```

Author: @favna u didn't want any screamy scream, right?

Reviewer 2: Yeah no exclamation mark @vladfrangu 👀
